### PR TITLE
added lambda layers docs for Python\Node (RK-3197)

### PR DESCRIPTION
**Pull request by BenTheKlein**

### CODE REVIEW

File: node-aws-lambda/README.md [hunk 1]
```diff
@@ -8,11 +8,22 @@ Before following this guide we recommend reading the basic [Node + Rookout] guid
 
 There are 3 simple steps to integrate Rookout into your existing Node application:
 
-1. Installing the Rookout SDK: `npm install --save rookout` and adding it in the entry file `const rookout = require('rookout/lambda');`
+1. Add our lambda layer, you can find the latest version [here](https://docs.rookout.com/docs/sdk-digests.html), make sure to replace the region and version placeholder with yours
```
Comment (LiranLast) [inline node-aws-lambda/README.md]: @MickaelAlliel @BenTheKlein can we somehow make it more updated? Maybe some badge?

File: node-aws-lambda/README.md [hunk 2]
```diff
@@ -8,11 +8,22 @@ Before following this guide we recommend reading the basic [Node + Rookout] guid
...
+    ```
+
+For further information check aws layers docs [here](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html)
+
+For alternative import, install the Rookout SDK run the following command: `npm install --save rookout`
 
 2. Wrapping your function with the Lambda wrapper as such :  
 
 ```javascript
+const rookout = require('rookout/lambda');
```
Comment (LiranLast) [inline node-aws-lambda/README.md]: Let's add options argument to the wrap in the example

File: node-aws-lambda/README.md [hunk 3]
```diff
@@ -39,23 +50,16 @@ exports.handler = rookout.wrap(handler);
                     --handler index.handler \
                     --runtime nodejs8.10 \
                     --environment Variables="{ROOKOUT_TOKEN=<Your Rookout Token>,ROOKOUT_ROOK_TAGS=lambda}" \
+                    --layers "ROOKOUT-LAYER-ARN" \
```
Comment (LiranLast) [inline node-aws-lambda/README.md]: What's this? Why layers in two different ways?

**Comment by BenTheKlein** [comment]
@LiranLast @yonitwis - the costumer from yesterday had no idea about layers - we need to re-think this doc

**Comment by LiranLast** [comment]
@BenTheKlein I think it's self-explanatory enough :/

File: node-aws-lambda/README.md (hunk 1 above)
Comment (MickaelAlliel) [reply node-aws-lambda/README.md]: A badge would require some work, AWS doesn't provide them.
the arn looks like `arn:aws:lambda:<YOUR_REGION>:032<YOUR_RUNTIME>5105219:layer:rookout_python<YOUR_RUNTIME>_v_0_1_74_1:1` and changes each version. No easy way to have it updated in the example here

File: node-aws-lambda/README.md [hunk 4]
```diff
@@ -8,11 +8,22 @@ Before following this guide we recommend reading the basic [Node + Rookout] guid
 
 There are 3 simple steps to integrate Rookout into your existing Node application:
 
-1. Installing the Rookout SDK: `npm install --save rookout` and adding it in the entry file `const rookout = require('rookout/lambda');`
+1. Add our lambda layer, you can find the latest version [here](https://docs.rookout.com/docs/sdk-digests.html), make sure to replace the region and version placeholder with yours  
+    
+    Example:
+    ```bash
+    aws lambda update-function-configuration --function-name YOUR-FUNCTION-NAME --layer arn:aws:lambda:us-east-1:032275105219:layer:rookout_python27_v_0_1_66_1:1
```
Comment (MickaelAlliel) [inline node-aws-lambda/README.md]: node here! not python

**Comment by BenTheKlein** [comment]
@MickaelAlliel done

**Pull request merged**